src/main/java/racing/Car.java

```diff
@@ -0,0 +1,39 @@
+package racing;
+
+public class Car {
+
+    private static final int MOVE_THRESHOLD = 4;
+    private static final int MAX_NAME_LENGTH = 5;
+    private final String name;
+    private int position;
+
+    public Car(String name) {
+        this(name, 0);
+    }
+
+    public Car(String name, int position) {
+        if (name.length() > MAX_NAME_LENGTH) {
+            throw new IllegalArgumentException("이름은 5자 이하만 가능합니다.");
+        }
+        this.name = name;
+        this.position = position;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void moveForward(int num) {
+        if (num >= MOVE_THRESHOLD) {
+            position++;
+        }
+    }
+
+    public int getPosition() {
+        return position;
+    }
```

Comment:
getter, setter ,toString와 같은 메서드들은 최하단에 위치시키고 다른 객체와의 협력을 위한 public 메서드들을 상위로 올려주시면 조금 더 코드를 파악하기가 쉬울 것 같아요. 😉

```suggestion
    public void moveForward(int num) {
        if (num >= MOVE_THRESHOLD) {
            position++;
        }
    }
    
    public String getName() {
        return name;
    }

    public int getPosition() {
        return position;
    }
```